auth/client/src/main/resources/schema/elytron-client-1_8.xsd

```diff
@@ -0,0 +1,1402 @@
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:sequence minOccurs="0" maxOccurs="unbounded">
+            <xsd:element name="configuration" type="authentication-configuration-type"/>
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="authentication-configuration-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                A configuration that can then be matched from the authentication rules.
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:all>
+            <xsd:element name="set-host" type="name-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        DEPRECATED. Override the destination host name for the authenticated call.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="set-port" type="port-number-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        DEPRECATED. Override the destination port number for the authenticated call.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="set-protocol" type="name-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        DEPRECATED. Override the protocol for the authenticated call.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element ref="abstract-user-spec" minOccurs="0"/>
+            <xsd:element name="set-mechanism-realm" type="optional-name-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        The name of the realm that will be selected by the SASL mechanism if required.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="rewrite-user-name-regex" type="regex-substitution-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        A regular expression pattern and replacement to re-write the user name used for authentication.
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="sasl-mechanism-selector" type="selector-type" minOccurs="0">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        A SASL mechanism selector using the syntax from org.wildfly.security.sasl.SaslMechanismSelector,fromString().
+                    </xsd:documentation>
+                </xsd:annotation>
+            </xsd:element>
+            <xsd:element name="http-mechanism-selector" type="name-type" minOccurs="0">
```

Comment:
@keshav-725 Just a minor, user might want to allow both DIGEST and BEARER for example. So we might want to configure multiple HTTP mechanisms here, so the name-type might not be the best here, maybe selector-type is better. 

Then the mechanism chosen by the client will be the mechanism that is specified in the server's challenge

Reply:
Just to make the new changes to the Elytron Client schema more clear, it's good practice to have one commit that just adds a new schema file for 1.8 that is identical to the previous version but just has the version changed. That commit should reference ELY-2575.

Then a subsequent commit can include the new elements/attributes your changes are adding. That can reference [ELY-201].